<launch>

	<!--param
		name="robot_description"
		textfile="$(find open_base)/urdf/description.urdf"
	/-->
	<param
		name="open_base2/robot_description"
		command="$(find xacro)/xacro '$(find open_base)/xacro/main2.urdf.xacro'"
	/>
	<!--param
		name="open_base/parameter/control/D"
		value="0.0"
	/>< PID -->
	<param
		name="open_base2/parameter/control/I"
		value="0.0005"
	/><!-- PID -->
	<param
		name="open_base2/parameter/control/P"
		value="5.0"
	/><!-- PID -->
	<param
		name="open_base2/parameter/initial/x"
		value="0.0"
	/>
	<param
		name="open_base2/parameter/initial/y"
		value="0.0"
	/>
	<param
		name="open_base2/parameter/initial/theta"
		value="0.0"
	/>
	<param
		name="open_base2/parameter/max/omega"
		value="50.0"
	/><!-- set by experiments (rad/s) -->
	<param
		name="open_base2/parameter/max/v"
		value="1.0"
	/><!-- set by experiments (m/s) -->
	<param
		name="open_base2/parameter/stop/angle"
		value="0.1047197551196597705355242034774843062905347323976457118988037109375"
	/><!-- pi / 30 -->
	<param
		name="open_base2/parameter/stop/distance"
		value="0.001"
	/>
	<param
		name="open_base2/parameter/wheel/radius"
		value="0.01905"
	/>

</launch>
